dbaas/workflow/steps/util/host_provider.py

```diff
@@ -198,6 +198,47 @@ def create_host(self, infra, offering, name, team_name, zone=None,
         host.save()
         return host
 
+    def prepare(self):
```

Comment:
what prepare means ?

Reply:
Prepare means that is a step before starting the creation of the host